README.md

```diff
@@ -29,6 +29,27 @@ Table of contents
 
 ## Getting Started
 
+### Build Prerequisites
+
+#### Go
+The current targeted version of the Go language runtime is v1.12
+
+#### ZeroMQ
```

Comment:
I would prefer to not duplicate this information here. Please change to reference the edgex-go README ZeroMQ section

Reply:
Changed to reference